subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/GeneratePluginDescriptors.java

```diff
@@ -58,11 +58,19 @@ public GeneratePluginDescriptors() {
         );
     }
 
-    @Internal("The relevant data is tracked via implementationClassById")
+    /**
+     *
```

Comment:
Can you add some Javadoc here? It is now empty.

Reply:
Oops, noted, fixed